cli/src/pcluster/config/cluster_config.py

```diff
@@ -1354,9 +1354,9 @@ def existing_fs_id_list(self):
         if self.shared_storage:
             for storage in self.shared_storage:
                 fs_id = None
-                if isinstance(storage, (SharedEfs, BaseSharedFsx)):
+                if isinstance(storage, (SharedEfs, SharedFsxLustre)):
                     fs_id = storage.file_system_id
-                elif isinstance(storage, SharedEbs):
+                elif isinstance(storage, (SharedEbs, ExistingFsxOpenZfs, ExistingFsxOntap)):
                     fs_id = storage.volume_id
```

Comment:
We are dealing with two concepts here: file systems and volumes. The fact that we are assigning `volume_id` to a variable that is called `fs_id` makes me think that we are kind of forcing our own logic. 
What about having at least the following renaming?
1. `fs_id` => `storage_id`
2. `fs_id_list` => `storage_id_list`

Reply:
Done